doc/source/user_guide/tutorials/animate/animate_time.rst

```diff
@@ -0,0 +1,361 @@
+.. _ref_tutorials_animate_time:
+
+======================
+Animate data over time
+======================
+
+.. |Examples| replace:: :mod:`ansys.dpf.core.examples`
+.. |Animator| replace:: :class:`Animator<ansys.dpf.core.animator.Animator>`
+.. |Field| replace:: :class:`Field<ansys.dpf.core.field.Field>`
+.. |FieldsContainer| replace:: :class:`FieldsContainer<ansys.dpf.core.fields_container.FieldsContainer>`
+.. |MeshedRegion| replace:: :class:`MeshedRegion <ansys.dpf.core.meshed_region.MeshedRegion>`
+.. |TimeFreqSupport| replace:: :class:`TimeFreqSupport <ansys.dpf.core.time_freq_support.TimeFreqSupport>`
+.. |animate| replace:: :func:`FieldsContainer.animate() <ansys.dpf.core.fields_container.FieldsContainer.animate>`
+.. |Result| replace:: :class:`Result <ansys.dpf.core.results.Result>`
+.. |Operator| replace:: :class:`Operator<ansys.dpf.core.dpf_operator.Operator>`
+.. |Workflow| replace:: :class:`Workflow<ansys.dpf.core.workflow.Workflow>`
+.. |Elemental| replace:: :class:`elemental<ansys.dpf.core.common.locations>`
+.. |ElementalNodal| replace:: :class:`elemental_nodal<ansys.dpf.core.common.locations>`
+.. |Nodal| replace:: :class:`nodal<ansys.dpf.core.common.locations>`
+.. |Faces| replace:: :class:`faces<ansys.dpf.core.common.locations>`
+.. |Overall| replace:: :class:`overall<ansys.dpf.core.common.locations>`
+.. |open_movie| replace:: :class:`pyvista.Plotter.open_movie`
+
+This tutorial demonstrates how to create 3D animations of data in time.
+
+:jupyter-download-script:`Download tutorial as Python script<animate_time>`  :jupyter-download-notebook:`Download tutorial as Jupyter notebook<animate_time>`
+
+To animate data across time, you must store the data in a |FieldsContainer| with a ``time`` label.
+
+
+Get the result files
+--------------------
+
+First, import a results file. For this tutorial, you can use the one available in the |Examples| module.
+For more information about how to import your own result file in DPF, see
+the :ref:`ref_tutorials_import_data` tutorial section.
+
+.. jupyter-execute::
+
+    # Import the ``ansys.dpf.core`` module
+    from ansys.dpf import core as dpf
+    # Import the examples module
+    from ansys.dpf.core import examples
+    # Import the operators module
+    from ansys.dpf.core import operators as ops
+    
+    # Define the result file path
+    result_file_path = examples.find_msup_transient()
+    # Create the model
+    model = dpf.Model(data_sources=result_file_path)
```

Comment:
I would avoid combining comments and imports. Instead, I feel it is more appropriate to have paragraphs and small code cells:

  ```suggestion
  
  Start by importing the `dpf`, `examples`, and `operators` modules:
  
  
.. jupyter-execute::

    from ansys.dpf import core as dpf
    from ansys.dpf.core import examples
    from ansys.dpf.core import operators as ops
    
Next, define the path to the result file and create the model:

.. jupyter-execute::
    
    result_file_path = examples.find_msup_transient()
    model = dpf.Model(data_sources=result_file_path)
```